sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -291,6 +298,12 @@ def _get_headers(headers, request_type, replica_count, uses_feature_flags, featu
     if replica_count > 0:
         correlation_context += ",ReplicaCount=" + str(replica_count)
 
+    if uses_load_balancing:
+        correlation_context += ",UsesLoadBalancing"
+
+    if failover_request:
+        correlation_context += ",FailoverRequest"
```

Comment:
We use "Failover" tag in .NET provider

https://github.com/Azure/AppConfiguration-DotnetProvider/blob/90835cf5efbfb26d6de6068e4c31c3050947b78a/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs#L35